spec/features/endpoints/forecast_spec.rb

```diff
@@ -1,9 +1,11 @@
 require 'rails_helper'
 
-RSpec.describe 'forecast endpoint' do
-  xit 'returns forecast info for a location' do
-    visit '/api/v1/forecast?location=denver,co'
+RSpec.describe 'forecast endpoint', type: :request do
+  it 'returns forecast info for a location' do
+    get '/api/v1/forecast?location=denver,co'
```

Comment:
without `type: :request` this was one of those 'oh crap, how do i test this?' moments that took a few minutes to figure out
`NoMethodError: undefined method `get' for #RSpec::`